lib/vfscore/Config.uk

```diff
@@ -22,6 +22,19 @@ help
 	Automatically mounts '/' during boot. The parameters are
 	compiled-in and cannot be changed at runtime.
 
+config LIBVFSCORE_NONLARGEFILE
```

Comment:
Sorry, I did not see this, can you move this option before `LIBVFSCORE_AUTOMOUNT_ROOTFS`? It is now splitting up the connected block of `LIBVFSCORE_AUTOMOUNT_ROOTFS` and `LIBVFSCORE_ROOTFS`, etc.

Reply:
Ah, yes, I missed that. Fixed.